tutorial/setup/external.md

```diff
@@ -11,5 +11,7 @@ At this point in time I do not have the resources to publicly host a PlanarAlly
 Important to know is that these services are **not** operated by me and thus I cannot guarantee that the software has not been altered with malicious intentions.
 
 ## Known providers
+*If you host or know someone who hosts PA publicly, feel free to let me know!*  
 
-At the time of writing only [dndbox](https://planarally.dndbox.com/) is known to me to offer PlanarAlly as a public service.  If you host or know someone who hosts PA publicly, feel free to let me know!
\ No newline at end of file
+[DnDbox.com](https://dndbox.com/) v0.19.3 is hosted in Switzerland/Europe and hosts a number of projects including Planar Ally support is offerred on [reddit](reddit.com/r/dndbox/)  
+[10leej.com](planarally.10leej.com) v0.19.3 hosted in Atlanta, Georgia/USA support provided by 10leej use contact information found on the [website frontpage](https://10leej.com/)  
```

Comment:
```suggestion
[10leej.com](https://planarally.10leej.com) v0.19.3 hosted in Atlanta, Georgia/USA support provided by 10leej use contact information found on the [website frontpage](https://10leej.com/)  
```

Reply:
also I just noticed the PA url on your frontpage has a misconfigured url ;)

Reply:
woops

Reply:
updated pull request (github frontpage is weird to me)